airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ads_insights_breakdowns.json

```diff
@@ -12,21 +12,53 @@
       "description": "The identifier for the mobile app associated with the ad.",
       "type": ["null", "string"]
     },
+    "body_asset_id": {
+      "description": "The ID of the main content or message of the ad.",
+      "type": ["null", "string"]
+    },
     "body_asset": {
       "description": "The main content or message of the ad.",
+      "type": ["null", "object"],
```

Comment:
One more random thought: we could maybe avoid the breaking change by adding a new field instead of re-using the same one. Something like `body_asset_object` and we can keep `body_asset` as `str(data["body_asset"])`. What I don't like about this solution though is that we start to drift from the API naming and it adds complexity in our code.

Reply:
It would be a still breaking change due to change in pk. Because we also changed pk from value of `str(data["body_asset"])` to `body_asset_id`.